File: app/src/main/java/com/example/msdc/api/TVResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class TVResult {
    @SerializedName("id")
    int id;

    @SerializedName("poster_path")
    String posterPath;

    @SerializedName("original_name")
    String name;

    @SerializedName("vote_average")
    double voteAverage;

    public double getVoteAverage() {
        return voteAverage;
    }

    public int getId(){ return id; }

    public String getPosterPath(){ return posterPath; }

    public String getName(){ return name; }
}

File: app/src/main/java/com/example/msdc/api/TVResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class TVResult (
    @field:SerializedName("id") val id: Int? = 0,
    @field:SerializedName("poster_path") val posterPath: String? = null,
    @field:SerializedName("original_name") val name: String? = null
) : Parcelable